app/controllers/users/sessions_controller.rb

```diff
@@ -50,5 +63,11 @@ def track_authentication_attempt(email)
 
       analytics.track_event('Authentication Attempt with nonexistent user')
     end
+
+    def password
+      params.require(:user)[:password]
+    rescue ActionController::ParameterMissing
+      ''
```

Comment:
Here's the test when password is missing: https://github.com/18F/identity-idp/pull/507/files#diff-696e7da55305238bfb1f076a429283a6R168

in my env, I got an `ActionController::ParameterMissing` exception when the `:user` param was missing. The scenario I am imagining is when the user agent is not our form, or our form, broken.